data/fields/curve.json

```diff
@@ -0,0 +1,9 @@
+{
+    "key": "curve",
+    "type": "combo",
+    "label": "Type",
+    "prerequisiteTag": {
+        "key": "hazard",
+        "value": "curve"
```

Comment:
The values of these keys use terms that may be unfamiliar to mappers who have only ever seen the symbol signs without any explanatory plaques beneath. Perhaps we could take the time to list out the documented values and also specify icons that depict the arrow on the sign.